configuration/backend_configuration/ampathformstranslations/hiv_death_form_translations_fr

```diff
@@ -0,0 +1,38 @@
+{
+  "uuid": "41af2def-841d-38b7-8d2e-df25bdd0b73f",
+  "form": "HIV Death Form",
```

Comment:
Hey @VeronicaMuthee, we don't have a form that's named `HIV Death Form`, but we do have the form `Death Form`. The `form` attribute must be provided with an existing form name for the translations to load successfully.